### PR TITLE
Adding support for GET /api/v1/vectors endpoint

### PR DESCRIPTION
**Pull request by naumraviz**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Python Version(s) Tested:
* Tenable.sc version (if necessary):

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by SteveMcGrath — CHANGES_REQUESTED** [review]

**Comment by SteveMcGrath** [comment]
For some reason the tests arent validating out the new code either...

```
tenable/apa/vectors/api.py                            22      9    59%   35-42, 129-139
```

https://github.com/tenable/pyTenable/actions/runs/9970892846/job/27550762173#step:5:1102

File: tenable/apa/vectors/api.py [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+    _next_token: str = None
+    _payload: Dict
+
+    def _get_page(self) -> None:
+        """
+        Request the next page of data
+        """
+        payload = copy(self._payload)
+        print(self._next_token)
```
[the remark this answers is not in the document]
Comment (naumraviz) [reply tenable/apa/vectors/api.py]: Hey 
In this endpoint we only support offset based pagination for now-  I fixed the method

File: tenable/apa/vectors/api.py [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+                 Filters to allow the user to get
+                 to a specific subset of Findings.
+                 For a more detailed listing of what filters are available,
+                 please refer to the API documentation
+                 linked above, however some examples are as such:
+
+                 - ``{"operator":"==", "key":"state", "value":"open"}``
+                 - ``{"operator":">",
+                      "key":"last_updated_at",
+                      "value":"2024-05-30T12:28:11.528118"}``
```
[the remark this answers is not in the document]
Comment (naumraviz) [reply tenable/apa/vectors/api.py]: Please review this again

File: tenable/apa/vectors/api.py [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+             >>> findings = tapa.findings.list()
+             >>> for f in findings:
+             ...     pprint(f)
+
+         Examples:
+             >>> tapa.findings.list(
+             ...     limit='10',
+             ...     sort_filed='last_updated_at',
+             ...     sort_order='desc',
+             ...     filter='value',
```
[the remark this answers is not in the document]
Comment (naumraviz) [reply tenable/apa/vectors/api.py]: Done

**Comment by naumraviz** [comment]
@SteveMcGrath Coverage looks fine now 
https://github.com/tenable/pyTenable/actions/runs/10036435064/job/27734176187?pr=818

**Review by SteveMcGrath — APPROVED** [review]

**Pull request merged**